content/en/docs/reference/glossary/container-runtime.md

```diff
@@ -17,5 +17,5 @@ tags:
 
 Kubernetes supports several container runtimes: {{< glossary_tooltip term_id="docker">}},
 {{< glossary_tooltip term_id="containerd" >}}, {{< glossary_tooltip term_id="cri-o" >}},
-and any implementation of the [Kubernetes CRI (Container Runtime
```

Comment:
Please avoid revising two different localizations in the same pull request. Would you be willing to omit this part of the change?